ports/esp32/Makefile

```diff
@@ -29,8 +29,12 @@ BUILD ?= build-$(BOARD)
 endif
 
 # Device serial settings.
-PORT ?= /dev/ttyUSB0
-BAUD ?= 460800
+ifneq ($(PORT),)
+PORT_ARG := -p $(PORT)
+endif
+ifneq ($(BAUD),)
+BAUD_ARG := -b $(BAUD)
+endif
```

Comment:
So this now leaves the selection of port and baud to `idf.py`?  Does that give a better user experience?

Reply:
As a default, yes. Esptool does have some auto detection logic that has worked very well for me during development.